exams/tasks.py

```diff
@@ -135,3 +129,18 @@ def export_exam_authorizations(self):
                 id__in=valid_auth_ids).update(status=ExamAuthorization.STATUS_IN_PROGRESS)
         except:  # pylint: disable=bare-except
             log.exception('Unexpected exception updating ExamProfile.status')
+
+
+@async.task
+def batch_process_pearson_zip_files():
+    """
+    Fetch zip files from pearsons sftp periodically.
+    """
+    try:
+        with sftp.get_connection() as sftp_connection:
+            processor = download.ArchivedResponseProcesser(sftp_connection)
+            processor.process()
+    except ImproperlyConfigured as exc:
+        log.warning('batch_process_pearson_zip_files got an ImproperlyConfigured exception: %s', exc)
```

Comment:
This task will be run on cron, my thought was to use a warning because it might not want exceptions coming up in all environments, thoughts?